metadata.rb

```diff
@@ -10,15 +10,11 @@
 
 supports 'ubuntu'
 
-depends 'apt'
 depends 'apache2'
-depends 'bind'
+depends 'bind', '= 1.3.0'
```

Comment:
You really want to avoid pins like this. You'll end up with a cookbook that can't be depsolved due to pins in other cookbooks >= is your friend. Let the users control pins aggressively if they need to.

Reply:
its specific during my rewrite phase because any version > than that exact version removes attributes and then its no longer compatible with this cookbook that depends on them. slowly converting the attributes to be compatible with the custom resource driven newer versions. Apache was a whirlwind to sort out. This cookbook was written for like 5 major versions of every dependency ago when attributes were king.

TLDR; plan is to remove all pins, they are just there to help get the ball rolling.